networkx/algorithms/connectivity/tests/test_edge_augmentation.py

```diff
@@ -285,120 +282,108 @@ def _augment_and_check(
         except nx.NetworkXPointlessConcept:
             orig_k = 0
     info = {}
+    if avail is not None:
+        # ensure avail is in dict form
+        avail_dict = dict(zip(*_unpack_available_edges(avail, weight=weight)))
+    else:
+        avail_dict = None
     try:
-        if avail is not None:
-            # ensure avail is in dict form
-            avail_dict = dict(zip(*_unpack_available_edges(avail, weight=weight)))
+        # Find the augmentation if possible
```

Comment:
Here is a collection of many tests where it is hard to see what happened if one fails. 
The print statements (and even an extra kwarg on the test function!) seem like they'd be helpful. But I suspect if we reorganized the tests using parameterization, etc they wouldn't be needed. Still, is it bette to have them there in case a test fails? At least to know which test it was...

Reply:
I have a hard time following what is happening in these tests - it does indeed seem like this might need some deeper investigation. FWIW part of the motivation for removing print statements is that I'm not even sure how/whether `pytest` presents info from `stdout` during test runs, so from my perspective these are mostly no-ops anyways (though there may be a set of flags/conditions under which the printed info is visible - I'm certainly no expert!).

I'm going to go ahead and back this one out too so it can be addressed separately!

Reply:
I use print statements regularly with pytest to debug what is going on. The test fails; and the failure report is shown with a section that shows what I believe is `stdout` (but might also have `stderr`?)  There might be better ways to debug a test failure. But I don't know how I'd do it without this. Do you have good tools to figure out why a test is failing?

Reply:
> But I don't know how I'd do it without this. Do you have good tools to figure out why a test is failing?

Nope, I've always just relied on the pytest logs - while that's sufficient for small, granular tests I can see how it would be limiting when calling these big "helper" functions. I suspect that (if anything) this file would benefit more from a deeper look/refactor attempt, so I'll leave it backed out!